pkg/operation/botanist/addons.go

```diff
@@ -379,6 +383,19 @@ func (b *Botanist) generateCoreAddonsChart() (*chartrenderer.RenderedChart, erro
 		return nil, err
 	}
 
+	// The node-local-dns interface cannot bind the kube-dns cluster IP since the interface
+	// used for IPVS load-balancing already uses this address.
+	if b.Shoot.IPVSEnabled() {
+		nodeLocalDNSConfig["clusterDNS"] = b.Shoot.Networks.CoreDNS.String()
+	} else {
+		nodeLocalDNSConfig["dnsServer"] = b.Shoot.Networks.CoreDNS.String()
+	}
```

Comment:
See comment above, there is not even a difference with these values

Reply:
Which comment, this is not the same for the kubelet, this is for the chart. 

The have the same value, but there place in the chart is different, clusterDNS is not set for Kube-proxy and is replaced by the node-local-dns logic `__PILLAR__CLUSTER__DNS__`.

Reply:
Ah, yep, right, thanks for pointing it out